plugins/link/dialogs/link.js

```diff
@@ -983,6 +1014,21 @@
 			}
 		};
 	} );
+
+	function validateTelNumber() {
+		var editor = this.getDialog()._.editor,
+			regExp =  editor.config.linkTelNumberValidate_regExp,
+			msg = editor.config.linkTelNumberValidate_msg,
+			dialog = this.getDialog();
```

Comment:
If we move this variable definition earlier, we could use it to get `editor`:

```javascript
editor = dialog._.editor
```

Microoptimisations – FTW!